core/driver/sources/src/main/scala/org/virtuslab/ideprobe/IntelliJFixture.scala

```diff
@@ -58,35 +64,39 @@ final case class IntelliJFixture(
   }
```

Comment:
The method withPlugin is broken now as `plugins` is unused

Reply:
Fixed.